typha/pkg/discovery/discovery.go

```diff
@@ -94,12 +100,14 @@ func DiscoverTyphaAddr(opts ...Option) (string, error) {
 
 	if options.addrOverride != "" {
 		// Explicit address; trumps other sources of config.
-		return options.addrOverride, nil
+		return Typha{
+			Addr: options.addrOverride,
```

Comment:
If an override is supplied, should there be an effort to resolve the `nodeName` for this?

Reply:
^^ genuine question .... I don't know and haven't checked.... suspect it won't ever be overridden though.